index.html

```diff
@@ -1,657 +1,1295 @@
+                                  <a class="cmp-image__link" href="/content/leaderimpact/lang/en2/resources/leading-through-a-crisis/video-series.html" data-cmp-hook-image="link">
+                                    <img src="./assets/latest-post4.png" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="">
+                                  </a>
+                                </div>
+                              </div>
+
+                              <div class="cmp-teaser__content">
+                                <h2 class="cmp-teaser__title">
+                                  <a class="cmp-teaser__title-link" href="/content/leaderimpact/lang/en2/resources/leading-through-a-crisis/video-series.html">Leading in Times of Crisis Video Series</a>
+                                </h2>
+                                <div class="cmp-teaser__description"><p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna.</p>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
                         </div>
-                    </a>
-                </div>
-                <div class="button resource-btn">
-                    <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                        <span class="btn-text">All Resources</span>
-                    </a>
+                      </div>
+                    </div>
+
+                    <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <a id="button-dbb21d03a5" class="cmp-button" href="/content/leaderimpact/lang/en2/resources.html">
+                        <span class="cmp-button__text">All Resources</span>
+                      </a>
+                    </div>
+                  </div>
                 </div>
+              </div>
             </div>
-        </section>
-        <section class="impact-story">
-            <div class="container">
-                <div class="stories-wrap">
-                    <div class="title">
-                        <div data-cmp-data-layer="" id="" class="title-wrap">
-                            <h2>Impact Stories</h2>
-                        </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid impact-story aem-GridColumn aem-GridColumn--default--12 layout-grid">
+          <div id="impact-container" class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 wrap-impact-story">

```

Comment:
`wrap-impact-story` can't go here.

Reply:
Why not just make the css target `#impact-container &:first-child` instead? This would mean less work for the authors.